compiler-plugin/src/main/kotlin/it/unibo/collektive/frontend/checkers/CheckersUtility.kt

```diff
@@ -153,4 +159,24 @@ object CheckersUtility {
             "$packageName.$functionName"
         }
     }
+
+    /**
+     * Returns a list of the arguments' types (in the form of [ConeKotlinType]) of the related function.
+     */
+    fun FirFunctionCall.getArgumentsTypes(): List<ConeKotlinType>? =
+        calleeReference
+            .toResolvedNamedFunctionSymbol()
+            ?.valueParameterSymbols
+            ?.map { parameter ->
+                parameter.resolvedReturnTypeRef.coneType
+            }
+
+    /**
+     * Checks whether if the called function accepts at least on argument of type
+     * [it.unibo.collektive.aggregate.api.Aggregate].
+     */
+    fun FirFunctionCall.hasAggregateArgument(): Boolean =
+        getArgumentsTypes()?.any {
+            it.classId == ClassId.fromString(AGGREGATE_CLASS_FQ_NAME.replace(".", "/"))
```

Comment:
Can we do it better? Does `ClassId.fromString` not accept FQN classes?

What about:

```kotlin
ClassId.topLevel(FqName(AGGREGATE_CLASS_FQ_NAME))
```

Could you verify if it works? Additionally, we can introduce a `fun String.toFqNameUnsafe(): FqName` to easily convert a `String` into a `FqName`. What do you think?